src/lib/openjp2/image.c

```diff
@@ -48,8 +48,8 @@ opj_image_t* OPJ_CALLCONV opj_image_create(OPJ_UINT32 numcmpts,
         image->color_space = clrspc;
         image->numcomps = numcmpts;
         /* allocate memory for the per-component information */
-        image->comps = (opj_image_comp_t*)opj_calloc(image->numcomps,
-                       sizeof(opj_image_comp_t));
+        image->comps = (opj_image_comp_t*)opj_calloc(1,
+                       image->numcomps * sizeof(opj_image_comp_t));
```

Comment:
Is this change necessary? I did keep the original code as much as possible, especially if the effect is the same.

Reply:
Not necessary; not sure why I changed this. What's the quick fix to revert back?